Add files via upload (#997)
algorand/docs: docs/run-a-node/setup/GivePermissionsRunAsAlgorandUser.md

title: Give Permissions to User to Run as Algorand User

> After installing with package manager for Debian or Red Hat distributions and want to customize a different user to be run algorand goal commands requires elevated permissions.

Reviewer: ```suggestion
After installing with package manager for Debian or Red Hat distributions you may want to customize a different user to be able to run algorand goal commands, which requires elevated permissions for that user.

```

To customize a different user, for example `algodev` user, then `algodev` user needs to be given permissions to run as `algorand` user. Change `algodev` username in this documentation to be specific to your environment.

---+++ Add file to /etc/sudoers.d/ Directory

---+++ Warning
Warning: Never edit this file with a normal text editor! Always use the visudo command instead!

Because improper syntax in the /etc/sudoers file can leave you with a broken system where it is impossible to obtain elevated privileges, it is important to use the visudo command to edit the file.

So from the example above `algodev`will be given elevated permissions to run commands as algorand user.

First, check what editor is saved as visudo editor.

On ubuntu only:
``` 
sudo update-alternatives --config editor
``` 

Check the directory first before making any changes. Confirm which files are there already. Confirm after edit too.
``` 
ls -la /etc/sudoers.d/
``` 

> Then vi a new file named algodevuser

Reviewer: ```suggestion
Then edit a new file named algodevuser

```

> The name `algodevuser` is arbitrary, but to give relevancy to you later. Change as necessary.

Reviewer: ```suggestion
The name `algodevuser` is arbitrary. Change as appropriate to your user.

```
